### PR TITLE
Add support for PROFILE= arg in BED_MESH_CALIBRATE

### PR DESCRIPTION
**Pull request by kmarty**
add parameter PROFILE= to be able use other than "default" bed mesh name.

### CODE REVIEW

File: Configuration/Adaptive_Mesh.cfg [hunk 1]
```diff
@@ -35,6 +35,7 @@ variable_detach_macro: 'Dock_Probe'     # Here is where you define the macro tha
 variable_display_parameters: True       # Display macro paramters in the console, useful for debugging the SETUP_KAMP_MESHING call, or more verbosity.
 
 gcode:
+    {% set mesh_profile_name = params.PROFILE|default('default') %}
```
Comment (ammmze) [inline Configuration/Adaptive_Mesh.cfg:38]: Any chance we could make it default to a variable we add above and then update the `SETUP_KAMP_MESHING` to set that default?

Comment (kmarty) [reply Configuration/Adaptive_Mesh.cfg:38]: Well, the problem is, the `BED_MESH_CALIBRATE` is quite often used with parameter `PROFILE=` and move profile name definition to the `SETUP_KAMP_MESHING` breaks this "backward" compatibility.
Anyway, yes it is absolutely possible. Give me a moment

Comment (jwhite) [reply Configuration/Adaptive_Mesh.cfg]: There is no place to report issues.  I have an issue where multi-material prints (like with the ERCF) don't use the correct bounds as the purge tower and priming lines now intersect with where the KAMP does it's purge line, creating a nozzle collision.  Any chance this pull request does something to fix that?

Comment (kmarty) [reply Configuration/Adaptive_Mesh.cfg]: @jwhite How is it related with this pull request?

Comment (jwhite) [reply Configuration/Adaptive_Mesh.cfg]: @kmarty Thanks.  There is no way to report an issue or to talk to the author, other than this message thread.

**Comment by dougshell** [comment]
totally not the right place, but i see no way to file an "issue" I am finding that sometimes the previous object size is retained and g80 will only mesh in the existing object perimeter.  Can this be fixed. If there is a better place for this question, let me know and I will close this.

**Comment by kyleisah** [comment]
I will not be adding profile support for this. There's no point. Adaptive meshes are meant to be taken every print, and can be saved from the web frontend (mainsail/fluidd). Thank you anyway!

**Comment by kmarty** [comment]
It was meant to seamless integration on RatOS (which uses profile named "ratos", not "default"), but nevermind. I switched to their adaptive meshing anyway (fewer obstructions).

**Pull request closed without merging**